src/annotator/guest.js

```diff
@@ -357,12 +356,7 @@ export default class Guest extends Delegator {
         return Promise.resolve({ annotation, target });
       }
 
-      // Find a target using the anchoring module.
-      const options = {
-        ignoreSelector: IGNORE_SELECTOR,
-      };
-      return this.anchoring
-        .anchor(root, target.selector, options)
+      return htmlAnchor(root, target.selector)
```

Comment:
When viewing a PDF the `anchoring` property gets overridden in [src/annotator/plugin/pdf.js](https://github.com/hypothesis/client/blob/129d1ba341736bd5c1848d8e3f94c19fd6ed5463/src/annotator/plugin/pdf.js#L44). Therefore the `Guest` class needs to continue using `this.anchoring` for the moment.